### ISSUE TITLE
Suggestion - Why not basing Waterfox Classic on UXP?

### ISSUE THREAD

**Issue by Sa-Ja-Di** [MrAlex94/Waterfox-Old#201]
Waterfox Classic is a dead-end - that should everyone see already.

But why not re-branding UXP based browsers as Waterfox Classic? The Pale Moon/UXP code is much more capable of viewing modern web pages than the old classic Firefox 56 engine, and old add-ons still are supported there too.

Additionally, it provides an ongoing strong effort now to implement missing ECMAScrip features. For example, right now work is happening to make BigInt possible in Pale Moon.

Therefor i highly suggest in breathing new life in Waterfox Classic and restarting it with using UXP as base code.

Win-Win situation for everyone!

**Comment by tordenflesk** [comment]
You'll lose WebExtensions, no?

**Comment by aeiouaeiouaeiouaeiouaeiouaeiou** [comment]
I had the same idea a few years ago until [this horrible thing happened](https://old.reddit.com/r/opensource/comments/pf6hyd/pale_moon_developers_abuse_mozilla_public_license/). Even though the situation has changed I still can't forget the [threats](https://github.com/WaterfoxCo/Waterfox-Classic/assets/69355938/55e45653-51ee-4e63-a618-2c2fd52b527b) to third party developers.

I'm really sorry.

**Comment by michael-j-oconnor** [comment]
> You'll lose WebExtensions, no?

Basilisk doesn't have then, so guessing not.

**Comment by hawkeye116477** [comment]
Yeah, you'll lose WebExtensions and also I don't know what the point of that would be, there is already browser with similar UI called Basilisk.
